ztp/ran-site-plan-crd/site-config-crd.yaml

```diff
@@ -0,0 +1,298 @@
+                              type: string
+                      clusterNetwork:
+                        description: ClusterNetwork is the list of IP address pools for pods. Default is 10.128.0.0/14 and a host prefix of /23.
+                        type: array
+                        items:
+                          description: |
+                            ClusterNetworkEntry is a single IP address block for
+                            pod IP blocks. IP blocks are allocated with size 2^HostSubnetLength.
+                          type: object
+                          required:
+                            - cidr
+                          properties:
+                            cidr:
+                              description: CIDR is the IP block address pool.
+                              type: string
+                            hostPrefix:
+                              description: |
+                                HostPrefix is the prefix size to allocate to each
+                                node from the CIDR. For example, 24 would allocate
+                                2^8=256 adresses to each node. If this field is
+                                not used by the plugin, it can be left unset.
+                              type: integer
+                              format: int32
+                      machineNetwork:
+                        description: MachineNetwork is the list of IP address pools for machines.
+                        type: array
+                        items:
+                          description: MachineNetworkEntry is a single IP address block for node IP blocks.
+                          type: object
+                          required:
+                            - cidr
+                          properties:
+                            cidr:
+                              description: CIDR is the IP block address pool for machines within the cluster.
+                              type: string
+                      serviceNetwork:
+                        description: |
+                          ServiceNetwork is the list of IP address pools for services default is 172.30.0.0/16.
+                          currently only one entry is supported.
+                        type: array
+                        maxItems: 1
+                        items:
+                          type: string
+                      apiVIP:
+                        description: APIVIP is the virtual IP used to reach the OpenShift cluster's API.
+                        type: string
+                      ingressVIP:
+                        description: IngressVIP is the virtual IP used for cluster ingress traffic.
+                        type: string
+                      manifestsConfig:
+                        description: day0 required manifestsConfig.
+                        type: object
+                        properties:
+                          ntpServer:
```

Comment:
Is this a list of servers ?

Reply:
I think it should be just one item based on the example below 
https://github.com/redhat-ztp/ztp-cluster-deploy/blob/master/create-cluster-sno/roles/chrony_day0/templates/chrony.conf#L2

if it is a list will remove it but if it is one item then can we consider the first item from the additionalNTPSources list below ?

Reply:
I think the desired input from a user would be a single list of NTP servers, and then we would just take the 1st one to put in the chrony.conf - Is that right @browsell ?

Reply:
Do we need this at all ? How is this different then L174 ?

Reply:
this is single server, L174 is a list. will remove it and consider the first item in the L174 list.

Reply:
Should this be removed?